unit-tests/src/test/java/org/eclipse/collections/impl/set/immutable/primitive/AbstractImmutableByteHashSetTestCase.java

```diff
@@ -52,6 +58,40 @@ protected MutableSet<Byte> newObjectCollectionWith(Byte... elements)
         return UnifiedSet.newSetWith(elements);
     }
 
+    protected static ByteArrayList generateCollisions()
```

Comment:
This one we can't easily avoid though as it's outside of the autogen classes & there's no generateCollisions1() for ByteArrayList - shall we leave like this?